index.js

```diff
@@ -211,7 +211,8 @@ class TestingFramework {
             let isValid = validation(expected, actual, key);
 
             if (!isValid) {
-                throw Error(`FAILED VERIFICATION:\nKEY: ${key}\nEXPECT: ${JSON.stringify(expected)}\nACTUAL: ${JSON.stringify(actual)}`);
+                console.log(`FAILED VERIFICATION:\nKEY: ${key}\nEXPECT: ${JSON.stringify(expected)}\nACTUAL: ${JSON.stringify(actual)}`);
```

Comment:
Using the stderr might be a better option
```suggestion
                console.error(`FAILED VERIFICATION:\nKEY: ${key}\nEXPECT: ${JSON.stringify(expected)}\nACTUAL: ${JSON.stringify(actual)}`);
```